hubzero/hubzero-cms
# Add see more courses button to intro page

What is the status of the PR?

[X] Ready.
[ ] Work in Progress.
[ ] Hold.

Are there any database schema changes?

[ ] Yes.
[X] No.

Description:

* Resolves STEMEd ticket #734 https://stemedhub.org/support/ticket/734
* Removes course limit in controller, which returns all courses to intro page
* Adds 'see more courses' link to intro page after displaying 12 courses
* Adds 'hyper-button' style to intro.css used by 'see more courses' link

Related tickets:

STEMEd#734 https://stemedhub.org/support/ticket/734

Testing instructions:

* Verify that 'see more courses' link does not appear at the bottom of the courses intro page (https://devhub.localdomain:5443/courses) when 12 or fewer than courses have been registered.
* Create more than 12 courses (must be admin: https://devhub.localdomain:5443/administrator/index.php?option=com_courses&controller=courses) and verify that the 'see more courses' button appears at the bottom of the intro page
